src/main/java/frc/robot/commandGroups/MidScoring.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commandGroups;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.ArmConstants;
+import frc.robot.subsystems.arm.commands.SetArmsPosition;
+import frc.robot.subsystems.gripper.Gripper;
+import frc.robot.subsystems.vision.Limelight;
+import frc.robot.subsystems.vision.VisionConstants;
+
+public class MidScoring extends SequentialCommandGroup {
+    private final Limelight limelight = Limelight.getInstance();
+    private final Gripper gripper = Gripper.getInstance();
+    public MidScoring(){
+        addCommands(
+                new ConditionalCommand(
+                        new TapeCommandGroup(),
+                        new AprilTagCommandGroup(),
+                        () -> limelight.getPipeline() == VisionConstants.REFLECTIVE_TAPE_PIPELINE
```

Comment:
It's better to have only one excellent command than two semi-excellent commands.